File: lib/f16_fortran/src/adc.f
      SUBROUTINE ADC(VT,ALT,AMACH,QBAR)
      DATA R0/2.377E-3/
      TFAC = 1.0 - 0.703E-5 * ALT
      T = 519.0 * TFAC
      IF (ALT .GE. 35000.0) T= 390.0
      RHO = R0 * (TFAC**4.14)
      AMACH= VT/SQRT(1.4*1716.3*T)
      QBAR = 0.5*RHO*VT*VT
C     PS = 1715.0 * RHO * T
      RETURN
      END

File: src/f16/adc.jl
function atmosphere(alt)
    # alt (ft)
    tfac = 1.0 - 0.703e-5 * alt
    # temperature (Rankine)
    alt >= 35000.0 ? T = 390.0 : T = 519.0 * tfac
    # density (slug / ft^3)
    ρ = R0 * tfac^4.14
    # sound velocity (ft/s)
    a = sqrt(1.4 * 1716.3 * T)
    # pressure (psf)
    p = 1715.0 * ρ * T
    return T, ρ, a, p
end


function adc(vt, T, ρ, a, p)

    amach = vt / a  # mach number
    qbar = 0.5 * ρ * vt^2  # dynamic pressure

    return amach, qbar

end
